app/views/admin/events/edit.html.haml

```diff
@@ -36,9 +36,18 @@
           .large-3.columns
             = f.association :venue, input_html: { data: { placeholder: 'Select sponsors' }}, required: true
         .row
-          .large-6.columns
+          .large-3.medium-6.small-12.columns
+            %br
+            = f.association :sponsors, input_html: { data: { placeholder: 'Select standard sponsors' }}, collection: Sponsor.all
+          .large-3.medium-6.small-12.columns
+            %br
+            = f.association :bronze_sponsors, input_html: { data: { placeholder: 'Select bronze sponsors' }}, collection: Sponsor.all
+          .large-3.medium-6.small-12.columns
+            %br
+            = f.association :silver_sponsors, input_html: { data: { placeholder: 'Select silver sponsors' }}, collection: Sponsor.all
+          .large-3.medium-6.small-12.columns
             %br
-            = f.association :sponsors, input_html: { data: { placeholder: 'Select sponsors' }}
+            = f.association :gold_sponsors, input_html: { data: { placeholder: 'Select sponsors' }}, collection: Sponsor.all
```

Comment:
Is it as easy to have a list of sponsors that each have a dropdown for a level? I think that's more intuitive for me, though it's absolutely not a blocker